PowerShellEditorServices.build.ps1

```diff
@@ -91,7 +91,7 @@ task SetupDotNet -Before Clean, Build, TestHost, TestServer, TestE2E {
 
         # Download the official installation script and run it
         $installScriptPath = "$([System.IO.Path]::GetTempPath())dotnet-install.$installScriptExt"
-        Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/cli/v$script:RequiredSdkVersion/scripts/obtain/dotnet-install.$installScriptExt" -OutFile $installScriptPath
+        Invoke-WebRequest "https://raw.githubusercontent.com/dotnet/sdk/master/scripts/obtain/dotnet-install.$installScriptExt" -OutFile $installScriptPath
```

Comment:
this needs the RequiredSdkVersion in it doesn't it?

Reply:
The script no longer exists in each branch. It takes a version argument below. I'm not sure why we took it from the tag each time, but that doesn't work anymore